library/build.gradle

```diff
@@ -30,7 +30,7 @@ androidExtensions {
 }
 
 dependencies {
-    implementation 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.4'
+//    implementation 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.4'
```

Comment:
Are we not pushing this to Bintray? Or is it working without this?

Reply:
Its working without it. It was not working with it.

Reply:
Its uploaded as v0.12